assets/stylesheets/test-details.scss

```diff
@@ -414,4 +414,16 @@ ul.modcategory {
     .fa-download:hover {
         opacity: inherit;
     }
+}
+
+.test-result-overview-link {
+    top: -0.1em;
+    position: relative;
+    left: 0.2em;
+    opacity: 0.8;
+    font-size:medium;
+    i {
+        color: #446E9B;
+        text-decoration: underline dotted;
```

Comment:
Ok, now that you've moved these lines here they make a difference. However, I still thinks the text decoration looks bad. You can keep the color if you like (I personally like to keep font awesome's default) but then please don't repeat the color code but use `$link-color` instead. (Btw, these colors are defined in `openqa_theme.scss`.)

Reply:
Done